scss/civicrm/common/_base.scss

```diff
@@ -1,17 +1,22 @@
 /* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
 
-body {
-  // Ensures that there is the same top padding regardless whether
-  // the user can see the admin toolbar or not
-
-  &:not(.toolbar) {
-    padding-top: $crm-main-menu-height !important;
-  }
+#{$outside-namespace-marker} {
```

Comment:
is this variable declared somewhere? I don't see it in current PR and seems newly introduces.

Reply:
yes, it's declared here https://github.com/civicrm/org.civicrm.shoreditch/blob/master/scss/bootstrap/overrides/_variables.scss#L1